src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/idam/IdamOpenId.java

```diff
@@ -0,0 +1,225 @@
+package uk.gov.hmcts.reform.judicialapi.idam;
```

Comment:
Copied over from `rd-commons-lib`

Reply:
why we need this class to be copied from rd-commons-lib, as we are already configured rd-commons-lib in build.gradle file

Reply:
Because that lib is not up to date and otherwise it would be blocking me. I removed that lib from build.gradle file.

Reply:
Ok..